### DISCUSSION TITLE
start_hppt.bat not working

### DISCUSSION THREAD

**Discussion by Pstross12** [w-okada/voice-changer#960] [Q&A]
When I try to open start_hppt.bat the cmd opens for a second or so and closes. After that it closes after 0,2 seocnds. Tried to find the solution on the internet but couldn't find it.

**Comment by YumeUwU** [comment]
me too

**Reply by navidcms2** [reply]
I don't have such problem
but when I Tried to run it as Administrator same thing happened
I don't know if that help
